e2e-saas/tests/JavaMaven.spec.ts

```diff
@@ -1,176 +0,0 @@
-            await projectTree.expandPathAndOpenFileInAssociatedWorkspace(fileFolderPath, tabTitle);
-            await editor.selectTab(tabTitle);
-        });
-
-        test('Java LS initialization', async () => {
-            await ide.waitStatusBarContains('Starting Java Language Server', 20000);
-            await ide.waitStatusBarTextAbsence('Starting Java Language Server', 1800000);
-            await ide.waitStatusBarTextAbsence('Building workspace', 360000);
-        });
-
-        test('Suggestion invoking', async () => {
-            await ide.closeAllNotifications();
-            await editor.waitEditorAvailable(tabTitle);
-            await editor.clickOnTab(tabTitle);
-            await editor.waitEditorAvailable(tabTitle);
-            await editor.waitTabFocused(tabTitle);
-            await editor.moveCursorToLineAndChar(tabTitle, 10, 20);
-            await editor.pressControlSpaceCombination(tabTitle);
-            await editor.waitSuggestion(tabTitle, 'append(char c) : PrintStream');
-        });
-
-        test('Error highlighting', async () => {
-            await editor.type(tabTitle, 'error', 11);
-            await editor.waitErrorInLine(11);
-            await editor.performKeyCombination(tabTitle, Key.chord(Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE));
-            await editor.waitErrorInLineDisappearance(7);
-        });
-
-        test('Autocomplete', async () => {
-            await editor.moveCursorToLineAndChar(tabTitle, 10, 11);
-            await editor.pressControlSpaceCombination(tabTitle);
-            await editor.waitSuggestionContainer();
-            await editor.waitSuggestion(tabTitle, 'System - java.lang');
-        });
-
-        test('Codenavigation', async () => {
-            await editor.moveCursorToLineAndChar(tabTitle, 9, 10);
-            await editor.performKeyCombination(tabTitle, Key.chord(Key.CONTROL, Key.F12));
-            await editor.waitEditorAvailable(codeNavigationClassName);
-        });
-
-    });
-
-    suite('Stop and remove workspace', async () => {
-        test('Stop workspace', async () => {
-            await dashboard.stopWorkspaceByUI(workspaceName);
-        });
-
-        test('Delete workspace', async () => {
-            await dashboard.deleteWorkspaceByUI(workspaceName);
-        });
-
-    });
-
-    async function runTask(task: string) {
-        await topMenu.selectOption('Terminal', 'Run Task...');
-        try {
-            await quickOpenContainer.waitContainer();
-        } catch (err) {
-            if (err instanceof error.TimeoutError) {
-                console.warn(`After clicking to the "Terminal" -> "Run Task ..." the "Quick Open Container" has not been displayed, one more try`);
-
-                await topMenu.selectOption('Terminal', 'Run Task...');
-                await quickOpenContainer.waitContainer();
-            }
-        }
-
-        await quickOpenContainer.clickOnContainerItem(task);
-    }
-});
```

Comment:
Is it just cleanup not related to the main issue with `7.7.1b` / PR, periodic checks

Reply:
Yes, this class is not needed anymore for the tests. Can I leave it here or should I create separate PR for it?

Reply:
yeah, let's leave it here